File: 240506/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim();
const array = ['apple', 'banana', 'grape', 'blueberry', 'orange'];
const result = array.filter(el => el[2] === input || el[3] === input);

console.log(result.join('\n') + '\n' + result.length);

File: 250106/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <iostream>
using namespace std;

int main() {
    string arr[5] = {"apple", "banana", "grape", "blueberry", "orange"};
    char input;
    int count = 0;
    cin >> input;

    for (int i = 0; i < 5; i++) {
        if (arr[i][2] == input || arr[i][3] == input) {
            cout << arr[i] << endl;
            count += 1;
        }
    }
    
    cout << count;
    return 0;
}